packages/components/src/summary-button/README.md

```diff
@@ -0,0 +1,42 @@
+# SummaryButton
```

Comment:
For new components, can we omit manual READMEs for now? We can set up a README auto-generation step like we started to do in `@wordpress/components` if necessary, but without it, it just adds two sources of truth which is a hassle to maintain and eventually consolidate. It would be great if we could just keep TSDocs as the source.

Reply:
Sounds good. Is there a plan though to add the auto generateed docs?

Reply:
On one hand I think a README isn't really necessary when you have IntelliSense docs and it also links out to a comprehensive docs site, but then again it might be useful for AI agents. So yes, pretty likely that we will add it.